controller/api/public/grpc_server.go

```diff
@@ -264,6 +269,33 @@ func (s *grpcServer) TapByResource(req *pb.TapByResourceRequest, stream pb.Api_T
 	}
 }
 
+// Pass through to Destination service
+func (s *grpcServer) Get(req *destinationPb.GetDestination, stream destinationPb.Destination_GetServer) error {
```

Comment:
similar to `client_test.go`, there's a `TestEndpoints` in `grpc_server_test.go`, consider re-implementing it.

Reply:
Given that `Get` in grpc_server.go is just a passthrough to `Get` in Destination's `server.go`, I think it might not be worth going through all the hoops to set up a test to hit that...